src/cascadia/LocalTests_TerminalApp/precomp.h

```diff
@@ -25,6 +25,8 @@ Author(s):
 #undef GetCurrentTime
 #endif
 
+#include <wil/win32_helpers.h>

```

Comment:
This is curious to me that this isn't needed in `LibraryIncludes`, and just adding it here works...

Reply:
Wasn't required to build & test. Feel free to include in future if necessary.

Reply:
I'm sorry, you added an include that _isn't required to build or test_?

... can we just delete it?

Reply:
Ohhhhhh ... duh ... yeah ... ahem ... it's been a long week.  Deleting now ;)

Reply:
Sorry, I'm unclear why this wil thing is still here and not in libraryincludes?

Reply:
RICH WHY IS THIS HERE AND NOT IN LIBRARYINCLUDES

Reply:
'CAUSE I WAS OOF LAST WEEK ON VACATION AND SWAMPED THE WEEK BEFORE BECAUSE I WAS GOING ON VACATION.

MR. SHOUTY! ;)

Reply:
Made the change. Plz. review.